web/src/Pages/TopicPage/BreadCrumbs/Breadcrumbs.jsx

```diff
@@ -1,73 +1,98 @@
 /* eslint-disable react/button-has-type */
 import { Link } from "react-router-dom";
-import { React } from "react";
+import { React, useEffect, useState } from "react";
 import PropType from "prop-types";
 import "../../../Styling/TopicBreadcrumbs/TopicBreadcrumbs.css";
 import { sendBreadCrumbEvent } from "../../../utils/gaEvents";
 
 function Breadcrumbs({ grandParent, parent, current }) {
+  const [dad, setDad] = useState(parent);
+  const [grandad, setGrandad] = useState(grandParent);
```

Comment:
Could perhaps use better names.
```suggestion
  const [daddy, setDaddy] = useState(parent);
  const [grandaddy, setGrandaddy] = useState(grandParent);
```

Reply:
ah, that is a good idea but i raise you:
```suggestion
  const [mommy, setMommy] = useState(parent);
  const [grandmommy, setGrandMommy] = useState(grandParent);
```

Reply:
love it